### ISSUE TITLE
No new DocType create menu after Nuget upgrade

### ISSUE THREAD

**Issue by craigs100** [umbraco/Umbraco-CMS#9972]
No new DocType create menu after Nuget upgrade

I am seeing this issue on Umbraco version: 8.12.0

Reproduction
------------

Initial web site was V8.9.1 with the Super Doc Type Create Menu package installed.
Removed the package and it's directories and DLL.
Upgraded site to V8.12.0 via Nuget
The upgrade process did not ask for permission to overwrite config files
The upgrade proceeded normally including the DB upgrade.

### Expected result

- Expected to see the Version declared as 8.12.0 
- Expected to see the new Doc Type Create menu

### Actual result

- Saw the Version declared as 8.12.0
- Saw the old version of the Doc Type Create menu

Tried the following in remediation without success:-

- Rebuilt solution
- Cleared cookies
- Logged out and in again
- Bumped CDF version
- Touched web.config
- Restarted app pool

**Comment by nul800sebastiaan** [comment]
From your "Actual result" it's clear the uninstall of Super Doc Type Create Menu didn't complete, have you uninstalled the nuget package for that as well?

I assume you uninstalled the package from the backoffice and that deleted the javascript file, but nuget will just keep copying the dll back in unless you uninstall it through nuget, so it injects menu items but the javascript is missing, so you don't get to create doctypes. 

Just did that same upgrade path and when the package is still installed I can create doctypes (each option works). When I uninstall the package using `Uninstall-Package` then I can still create document types. 

I'll close this for now, if that's not it then some more debugging is needed. My old blog post is still quite valid for that, although you already did many of the common things - https://cultiv.nl/blog/how-to-diagnose-umbraco-upgrade-problems/

**Comment by craigs100** [comment]
Hi Seb, That's not what happened. I unistalled it via Nuget. Then had to remove the DLL and relevant app_plugins directory. That was before the upgrade. The Nuget upgrade can't have copied over all the files as when I did that from the zip later, it all worked.

Now I'm having issues with the live version wanting to do an install instead of an upgrade, but that'll be a different issue if I find it's not my fault ;)
